doc/source/tutorial/stats/continuous_laplace_asymmetric.rst

```diff
@@ -0,0 +1,20 @@
+
+.. _continuous-laplace:
+
+Asymmetric Laplace Distribution
+================================================================
+
+.. math::
+   :nowrap:
+
+    \begin{eqnarray*} f(x, \kappa) &=& \frac{1}{\kappa+\kappa^{-1}}\exp(-x/\kappa),\quad x\ge0;\\
+                     &=& \frac{1}{\kappa+\kappa^{-1}}\exp(x\kappa),\quad x<0.\end{eqnarray*}
+
+.. math::
+   :nowrap:
+
+    \begin{eqnarray*} \mu &=& \kappa^{-1}-\kappa\\
+    \sigma^2 &=& \kappa^{-2}-\kappa^2\\
+    \tilde\mu_3 &=& \frac{2(1-\kappa^6)}{(1+\kappa^4)^{3/2}}\end{eqnarray*}
+
```

Comment:
Are there other values that can be listed?  doc/source/tutorial/stats/continuous.rst lists many about 15 functions or attributes.

Reply:
There are, but it gets complicated, and these are well-documented online. See e.g. https://en.wikipedia.org/wiki/Asymmetric_Laplace_distribution

Reply:
@pvanmulbregt Is this satisfactory? I added a suggestion to fix `\sigma^2` -to `\mu_{2}`,  `\tilde\mu_3` to `\gamma_{1}`, and I added `\gamma_{2}`.